mappings/net/minecraft/world/gen/surfacebuilder/NoiseMaterialSurfaceBuilder.mapping

```diff
@@ -0,0 +1,39 @@
+CLASS net/minecraft/unmapped/C_tyauowvy net/minecraft/world/gen/surfacebuilder/NoiseMaterialSurfaceBuilder
```

Comment:
Not sure if "noise material surface builder" really describes what this class does, it also has the steep slope things.
Is this class used for the stone shores as well? If yes, `AbstractMountainSurfaceBuilder` like Yarn doesn't really fit either.
Not sure.

Reply:
iirc it is used for stone shores.